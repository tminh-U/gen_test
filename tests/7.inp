14
15 17 17 5 4 6 12 9 12 18 17 6 15 5 
3
5 12
7 10
13 13
