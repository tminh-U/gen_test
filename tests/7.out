76
12
2
